deploy/deploy.yml

```diff
@@ -0,0 +1,50 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: kubernetes-external-secrets
+  labels:
+    app.kubernetes.io/name: kubernetes-external-secrets
+
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: kubernetes-external-secrets
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: kubernetes-external-secrets
+    spec:
+      containers:
+        - name: kubernetes-external-secrets
+          image: godaddy/kubernetes-external-secrets:1.3.0
+          envFrom:
+          - configMapRef:
+              name: external-secrets-config
+          # env:
```

Comment:
Uncomment this and include the ConfigMap?

Reply:
I have it locally - will clean up and push the changes